examples/ecs/system_piping.rs

```diff
@@ -56,6 +58,26 @@ fn handler_system(In(result): In<Result<usize, ParseIntError>>) {
     }
 }
 
+fn calc1() -> i32 {
```

Comment:
We should *probably* try an make these example a little more useful / representative. I'll chew on simple demos.

Reply:
My original use case was for a puzzle game where a couple of individual systems queried the puzzle to see if the puzzle was solved (think of checking rows and columns for a sudoku) and then return a boolean if the requirement was met.

The puzzle is solved if all the systems returned `true`. Technically this can also work with the `and` combinators, but if you also wanted to display to the user which condition wasn't met, those combinators wouldn't work and you'd need a specific handler system.